index.html

```diff
@@ -96,16 +97,57 @@ <h2>Multi-Post Stories</h2>
                             </div>
                         </div>
                     </div>
                 </div>
 
+                <!-- ############ POPUP WINDOW START HERE -->
+
+                <div class="modal" id="modal">
+                    <div class="text-container-row">
+                        
+                        <div class="container-desk">
+                            <div class="modal-title">
+                                <h2 class="mobile txt-modal">Multi-Post Stories</h2>
+                                <h2 class="hidden txt-modal">Keeping track of hundreds  of components website</h2>
+                                <button data-close-button class="close-btn">&times;</button>
+                            </div>
+                            <div class="row-skills">
+                                <ul>
+                                    <li>HTML</li>
+                                    <li>Bootstrap</li>
+                                    <li>Ruby on Rails</li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="grid-template">
+                            <a class="snap" href="#"><img src="images/Snapshoot Portfolio.png" alt="image popup"></a>
+                        <div class="grid">
+                            <div class="modal-text">
+                                <p class="mobile description-modal">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+
+                                <p class="hidden description-modal">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. <br> <br>
+
+                                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.</p>
+                            </div>
+                            <div class="buttons">
+                                <a href="#">See Live <img src="images/Icon.png" alt="icon" class="live"></a>
+                                <a href="#">See Source <img src="images/Icon -GitHub.png" alt="icon github" class="source"></a>
+                            </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
+                <div id="overlay"></div>
```

Comment:
- [ ] You've done an excellent job so far :+1:. However, as per the project requirement, the popup section should be created using JavaScript. Therefore, it is necessary to remove the popup section from your HTML file and implement it using JavaScript instead. Adhering to this requirement ensures consistency and follows the intended guidelines set for the project. Please proceed with removing the popup section from the HTML file and implementing it using JavaScript accordingly.